examples/pytorch/graphsage/train_sampling_multi_gpu.py

```diff
@@ -81,7 +81,7 @@ def run(proc_id, n_gpus, args, devices, data):
 
     train_mask = train_g.ndata['train_mask']
     val_mask = val_g.ndata['val_mask']
-    test_mask = ~(test_g.ndata['train_mask'] | test_g.ndata['val_mask'])
+    test_mask = test_g.ndata['test_mask']
```

Comment:
Does this work for other ogb dataset?

Reply:
As far as I can tell the accuracy is unaffected for `reddit` and `ogbn-products`.